imports/api/rest/post-process-api-payload-request.js

```diff
@@ -690,7 +692,10 @@ export default (req, res) => {
         }
       })
       return res.send(503, 'Processing similar payload')
-    } else if (!existingPayload.done) {
+    } else if (!existingPayload.done || hasExpired) {
+      if (hasExpired) {
+        logger.warn(`API payload ${requestId} has been processing for over 1 minute, so it is persumed to be in limbo and will be retried`)
```

Comment:
not sure what logger with are using, but in Go you would typically do a:

    log.WithField("requestId", requestId).Info("expired for processing for over a minute")

Reply:
This makes it possible to query on requestID